src/desktop/models/article.coffee

```diff
@@ -257,6 +257,7 @@ module.exports = class Article extends Backbone.Model
       "headline": @get('thumbnail_title')
       "url": @fullHref()
       "thumbnailUrl": @get('thumbnail_image')
+      "datePublished": @get('published_at')
       "dateCreated": @get('published_at')
```

Comment:
Both of these fields use the same underlying `published_at` attribute, just as in the existing toJSONLDAmp [implementation](https://github.com/artsy/force/blob/615ef1f6c00a4417a0d23ef81eaabb4c15f9c25d/src/desktop/models/article.coffee#L275-L276).